### ISSUE TITLE
[Bug] <title> Missing resume?

### ISSUE THREAD

**Issue by crimsonduelist** [AmruthPillai/Reactive-Resume#1709]
### Is there an existing issue for this?

- [X] Yes, I have searched the existing issues and none of them match my problem.

### Product Variant

Cloud (http://rxresu.me)

### Current Behavior

I assume there was some big update, cuz I seem to either be missing a resume or some of the changes affected the one I had?
Could a resume ve been deleted over an update that took place on your backend?
I saw some older issues like
[](https://github.com/AmruthPillai/Reactive-Resume/issues/761) -here I believe some changes caused loss of saved resumes but backup hosted on another link were provided
[](https://github.com/AmruthPillai/Reactive-Resume/issues/1579 ) -here it was just a matter of waiting for a migration to finish
Could it be a similar situation?

### What template are you using?

None

**Comment by pineapples5972** [comment]
Same problem happen with me. And its second time i noticed it.
First time I write my resume half way then i leave it that way and come back some days latter to finish it. 
I finish it and after a week when i visit it again it showing that unfinished resume. 
I thought it could be my browser problem or maybe because i logged in with github or google manually.
So i tried logout and loggin both way. It wasnt working. I gave up and visited after a week again to rewrite new resume thinking i lost my old what i see to my surprise my finished resume was there then i understand it might be a bug after re-edittng it i visited again after a week or so and now i am seeing that half unfinished resume again. not even the last i edited it. 
So i though i should report a bug.
Please fix it.

**Comment by CottonEyeJoee** [comment]
i don't know if this applies but my resumes are all gone but i can still find the pdf on the public pdf path. the only thing i did is adjust my email and maybe it has something todo with github too because there i updated my email too. and i use github login. it's not a complete lost but gont dangerously close to complete loss of my resume :/

**Comment by lelelemonade** [comment]
It happens to me too. But when I switch from chrome to tor browser, it fixed.

**Comment by amruthpillai** [comment]
Closing this issue as I'm doing some clean up to move to a newer version of the project. I apologize in advance for the inconvenience if you find the need to reopen this issue.

Please read #2499 for more information about the migration.

**Comment by veridynn** [comment]
@amruthpillai could you do me a favour and kill the dead account, i don't like the idea of my details open out there on the web 😬

**Comment by amruthpillai** [comment]
@veridynn of course. Could you send me an email to hello@amruthpillai.com of the link to your resume that you're able to publicly access? I can delete the account and all related data for you.

**Comment by veridynn** [comment]
@amruthpillai I tried emailing hello@amruthpilla.com but it bounced with a `DNS NXDOMAIN` error, so the domain doesn’t exist and the email is probably not set up anymore, same as with your website. 🙃

```
DNS Error: DNS type 'mx' lookup of [amruthpilla.com](http://amruthpilla.com/) responded with code NXDOMAIN Domain name not found: [amruthpilla.com](http://amruthpilla.com/) For more information, go to https://support.google.com/mail/?p=BadRcptDomain
```

**Comment by amruthpillai** [comment]
Oh, that's my mistake. I made a typo in spelling my own name. That's embarrassing 🙈 There's supposed to be an `i` at the end, just like my username here on GitHub.

The email is `hello@amruthpillai.com`.

**Comment by derickson2402** [comment]
Hello! Thank you for your continued efforts on this project. It has been very helpful as i apply for jobs.

Unfortunately, I logged in tonight and found that all my resumes were gone. I have PDF backups but no JSON backups. I can still access my resumes via public link. Is there any way to get the original JSON's back?

I believe I had logged in after January, after the migration, and edited a resume. There were formatting issues, but the resumes migrated over. So I'm not sure where they went this time, or why.

**Comment by amruthpillai** [comment]
> Hello! Thank you for your continued efforts on this project. It has been very helpful as i apply for jobs.
> 
> Unfortunately, I logged in tonight and found that all my resumes were gone. I have PDF backups but no JSON backups. I can still access my resumes via public link. Is there any way to get the original JSON's back?
> 
> I believe I had logged in after January, after the migration, and edited a resume. There were formatting issues, but the resumes migrated over. So I'm not sure where they went this time, or why.

Please share the public link with me on my email mentioned above, as well as the email and username of the account you are able to sign in to now.

I'll get this sorted for you.
